R/processInputsToXML.R

```diff
@@ -21,128 +21,121 @@ setGeneric(name="XML",def=function(stencil, fabric, knife){standardGeneric("XML"
 #'XML(sg, wd, webprocess())
 #'}
 #'@rdname XML-method
-#'@importFrom XML newXMLNode newXMLTextNode addChildren toString.XMLNode xmlChildren<- xmlValue<-
 #'@export
 setMethod(f = "XML",signature = c("ANY","webdata","webprocess"), 
           definition = function(stencil, fabric, knife){
             #stencil can be webgeom OR simplegeom 
             
-  knife <- .setProcessInputs(webprocess = knife, stencil = stencil, fabric = fabric)
-  top <- newXMLNode(name='wps:Execute',
-                    attrs=c('service'="WPS",'version'= version(knife),
-                            'xsi:schemaLocation' = paste(c(knife@WPS_NAMESPACE,knife@WPS_SCHEMA_LOCATION),collapse=" ")),
-                    namespaceDefinitions=c('wps' = knife@WPS_NAMESPACE,
-                                           'ows' = knife@OWS_NAMESPACE,
-                                           'ogc' = knife@OGC_NAMESPACE,
-                                           'xlink' = knife@XLINK_NAMESPACE,
-                                           'xsi' = knife@XSI_NAMESPACE))
-  
-  id	<-	newXMLNode("ows:Identifier",newXMLTextNode(knife@algorithm),parent=top)
-  di	<-	newXMLNode("wps:DataInputs",parent=top)
+  knife <- .setProcessInputs(webprocess = knife, 
+                             stencil = stencil, 
+                             fabric = fabric)
+  
+  whisker_list <- get_wps_execute_attributes(knife)
+  
+  whisker_list["identifier"] <- knife@algorithm[[1]]
+  
+  input_list <- list() # unnamed list for {{#inputs}}
   
   for (i in 1:length(knife@processInputs)){
-    postNm	<-	names(knife@processInputs[i])
-    postVl	<-	knife@processInputs[postNm]
-    if (!is.na(postVl)){
+    input_identifier	<-	names(knife@processInputs[i])
```

Comment:
bit of goofy whitespace here

Reply:
Yeah... this file has all kinds of tabs in it. Strange.